src/servies/__mocks__/api.js

```diff
@@ -0,0 +1,15 @@
+export const fetchRegions = async () => ['서울', '대전', '대구', '부산', '광주', '강원도'];
```

Comment:
```suggestion
export const fetchRegions = jest.fn(async () => ['서울', '대전', '대구', '부산', '광주', '강원도']);
```

같이 mock function 으로 선언해주셔야해요.